extending-valtimo/plugin/custom-plugin-definition.md

```diff
@@ -82,6 +82,61 @@ class TwitterPluginFactory(
 }
 ```
 
+### Plugin categories
+There are several use cases in which a plugin might rely on another.
```

Comment:
Missing an introduction on what plugin categories are. This section only answers "why" plugin categories are necessary, but doesn't state "what"  they are.